models/contributor.py

```diff
@@ -155,7 +155,7 @@ async def post_to_discord(self) -> None:
         webhook = DiscordWebhook(
             url=bot_settings.discord_hook,
             content="The top contributor of this month is "
-            + f"`{self.login}` with {self.pr_count} merged prs and {self.issue_count} opened issues.\n@everyone",
+            + f"`{self.login}` with {self.pr_count} merged prs and {self.issue_count} opened issues.",
```

Comment:
I suggest returning the `@everyone` mention as @diwash007 suggested:

```py
content=f"The top contributor of this month is `{self.login}` with {self.pr_count} merged PRs and {self.issue_count} opened issues.\n\n@everyone"
```

Reply:
@Omar8345 Well @thenishantsapkota discouraged this feature overall since it'd cause chaos if somebody uses it on a large Discord server.